src/Microsoft.Health.Abstractions/Features/Operations/IWorker.cs

```diff
@@ -0,0 +1,23 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Abstractions.Features.Operations
+{
+    /// <summary>
+    /// Abstraction for a worker
+    /// </summary>
+    public interface IWorker
```

Comment:
What's the point of this?

Reply:
It was intended to be used later by IPublishEvetntsWorker and hopefully other workers. But after re-checking based on your comment the usages in other workers realized I can get rid of this and instead move it to IPublishEvetntsWorker since each worker seems to have a different signature.